### PR TITLE
Bugfix lwdev 4677 client account exception

### PR DESCRIPTION
**Pull request by igromyko**

### CODE REVIEW

File: src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs [hunk 1]
```diff
@@ -497,30 +498,38 @@ private bool IsPending(KycStatusLogRecord row)
                 partnersDict[p.PublicId] = p.Name;
             }
 
+            Dictionary<string, ClientAccountInformationModel> clients = null;
+            if (items != null)
```
Comment (starkmsu) [inline src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs]: Add this check right after Items are retrieved with GetRecordsForPeriodAsync call.
If it's null then I assume from the code below - we can return null right away, right?

File: src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs [hunk 2]
```diff
@@ -497,30 +498,38 @@ private bool IsPending(KycStatusLogRecord row)
...
-                                if (client?.PartnerId != null)
-                                {
-                                    if (partnersDict.TryGetValue(client.PartnerId, out string name))
-                                    {
-                                        partnerName = name;
-                                    }
-                                    else
+                                ClientAccountInformationModel client;
+                                if (clients.TryGetValue(item.ClientId, out client)) {
+                                    if (client.PartnerId != null)
```
Comment (starkmsu) [inline src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs]: Better use string.IsNullOrWhiteSpace method here - empty string also should be ignored.

File: src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs (hunk 1 above)
Comment (igromyko) [reply src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs]: refactored

File: src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs (hunk 2 above)
Comment (igromyko) [reply src/Lykke.Service.KycReports.Services/Reports/KycReportingService.cs]: changed

**Review by starkmsu — APPROVED** [review]

**Pull request merged**